paddle/fluid/memory/allocation/allocator_facade.cc

```diff
@@ -42,6 +42,7 @@
 #ifdef PADDLE_WITH_XPU
 #include "paddle/fluid/platform/xpu_info.h"
 #endif
+#include "paddle/fluid/platform/npu_info.h"
```

Comment:
#ifdef PADDLE_WITH_ASCEND_CL

Reply:
I will refine this kind includes in the future, for this line, `paddle/fluid/platform/npu_info.h` is empty if not compiled with ASCEND_CL, so it is ok.